src/renderer/browser/actions/makeInvokeActions.js

```diff
@@ -4,8 +4,9 @@ import { isArray } from 'lodash';
 
 import { GAS } from 'shared/values/assets';
 
+import createScript from 'shared/util/createScript';
```

Comment:
:neckbeard: Whitespace not needed above this line since they're being imported from the same folder.